Sources/RInAppMessaging/Router.swift

```diff
@@ -335,9 +341,6 @@ internal class Router: RouterType, ViewListenerObserver {
     }
 
     private func updateFrame(targetView: UIView, tooltipView: TooltipView, superview: UIView, position: TooltipBodyData.Position) {
-        guard targetView.superview != nil else {
```

Comment:
Why this guard was removed?

Reply:
This targetView.superview will never be nil because it's already check in Line 215: 
```swift
let superview = self.findParentViewForTooltip(targetView)
```
When the superview is nil, it will return an error.